ganga/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py

```diff
@@ -15,6 +15,7 @@
 from GangaDirac.Lib.Backends.DiracUtils import get_result
 from GangaCore.GPIDev.Lib.GangaList.GangaList import GangaList, makeGangaListByRef
 from GangaCore.GPIDev.Adapters.IGangaFile import IGangaFile
+import GangaCore.GPI as GPI
```

Comment:
Looks odd. Why do we want to use the GPI? Isn't that only for the end user?

Reply:
I agree it is not idea. But `LHCbDataset` should exist as a standalone object so this was the only way I could think of to access a job on the registry.